.github/workflows/build.yml

```diff
@@ -7,17 +7,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.13
-      uses: actions/setup-go@v1
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
       with:
-        go-version: 1.13
+        go-version: 1.14
       id: go
 
     - name: Check out code into the Go module directory
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
 
     - name: Install golangci-lint
-      run: curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.21.0
```

Comment:
Can't we use the Github action as in the other projects? Maybe this would reduce the overhead a little:

https://github.com/metal-stack/metal-api/blob/master/.github/workflows/latest.yaml#L14-L17

Reply:
done